checker/src/org/checkerframework/checker/units/UnitsAnnotatedTypeFactory.java

```diff
@@ -282,8 +284,8 @@ private boolean isUnitsMultiple(AnnotationMirror metaAnno) {
     }
 
     /**
-     * Look for an @UnitsRelations annotation on the qualifier and add it to the list of
-     * UnitsRelations.
+     * Look for an {@link org.checkerframework.checker.units.qual.UnitsRelations} annotation on the
```

Comment:
Are the changes in this method related to this pull request or do they belong in a separate Units Checker pull request?

Reply:
In general I was cleaning up any loader related code, in this case minimizing the scope of the try/catch blocks to where the exceptions can be thrown. I'll move the changes here to another PR.

Reply:
Thanks!  It makes it much easier to review a pull request if it only contains changes related to a single goal.